python/basics/AmpLabs_Example_Basic_Plotting.ipynb

```diff
@@ -257,335 +298,174 @@
       },
       "outputs": [],
       "source": [
-        "user_token = \"<your-token>\"\n",
-        "cell_id = 'your-cell-id'"
+        "user_token = \"eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjV6am9vQkw5LUluRGRFTzl3UGVXMCJ9.eyJodHRwczovL3VzZXIuY29tL2VtYWlsIjoicmFtaW5ybXpAZ21haWwuY29tIiwiaXNzIjoiaHR0cHM6Ly9hdXRoLmFtcGxhYnMuYWkvIiwic3ViIjoibGlua2VkaW58X2diMlloZVpMOCIsImF1ZCI6WyJodHRwczovL2FtcGxhYnMuc2VydmVyIiwiaHR0cHM6Ly9kZXYtaDl0bWZ0ZjkudXMuYXV0aDAuY29tL3VzZXJpbmZvIl0sImlhdCI6MTY1ODE4MDkzOCwiZXhwIjoxNjU4MjY3MzM4LCJhenAiOiJzVGh1UTBMY0t2T1pqbUl4WUJDR3RjZmJ1cDdQcWJGciIsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwifQ.RlFZlsCVLg77SorYjSEyMSNcmgjHLcJvIFJPGWl1n_eiCbIcnrBLOG33Op6kJye2AjbiUvwyPFUAQ5kDZFsWoUdhlFwxYmioLJwb5liibnm571U1_iUNg4jL2AbcHOLO_hD3OFp_V5snABZHNnLPKNczaB0ctNsGtgJHqAWbduy7XoS3QBwMIAdfw5gN_sGDfDWShSihrhCtzFdFoU6FQzlwsNsfBOAVU9kMepKXVpHiqGRHW6tQkYx-9hF2s2o_Pf8wHGgbC2abV8gm4v6LJGi0dZpYgG07kLEyUzTLjTq2u5MDeae-u7DfskZW3WBRt9aEBdpVwpNNpHx-rIcZnQ\"\n",
```

Comment:
This token will point to your personal user. Do you intend to have users use your account?